sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentClassifierAdministrationLiveTests.cs

```diff
@@ -179,16 +179,16 @@ public async Task GetDocumentClassifiers(bool useTokenCredential)
         {
             var client = CreateDocumentModelAdministrationClient(useTokenCredential);
             var classifierIds = new string[] { Recording.GenerateId(), Recording.GenerateId() };
-            var description = "This classifier was generated by a .NET test.";
+            var description = $"This classifier was generated by a .NET test: {nameof(GetDocumentClassifiers)}";
 
             await using var disposableClassifier0 = await BuildDisposableDocumentClassifierAsync(classifierIds[0], description);
             await using var disposableClassifier1 = await BuildDisposableDocumentClassifierAsync(classifierIds[1], description);
 
             var idMapping = new Dictionary<string, DocumentClassifierDetails>();
             var expectedIdMapping = new Dictionary<string, DocumentClassifierDetails>()
             {
-                { disposableClassifier0.Value.ClassifierId, disposableClassifier0.Value },
-                { disposableClassifier1.Value.ClassifierId, disposableClassifier1.Value }
+                { classifierIds[0], disposableClassifier0.Value },
+                { classifierIds[1], disposableClassifier1.Value }
```

Comment:
Just changing the variables we're using for simplicity but they hold the same values (classifierIds[0] == diposableClassifier0.Value.ClassifierId).